spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/LogFile.java

```diff
@@ -39,13 +40,27 @@
 	 * The name of the Spring property that contains the name of the log file. Names can
 	 * be an exact location or relative to the current directory.
 	 */
-	public static final String FILE_PROPERTY = "logging.file";
+	public static final String FILE_PROPERTY = "logging.file.name";
 
 	/**
 	 * The name of the Spring property that contains the directory where log files are
 	 * written.
 	 */
-	public static final String PATH_PROPERTY = "logging.path";
+	public static final String PATH_PROPERTY = "logging.file.path";
+
+	/**
+	 * The name of the Spring property that contains the name of the log file. Names can
+	 * be an exact location or relative to the current directory.
+	 */
+	@Deprecated
+	public static final String DEPRECATED_FILE_PROPERTY = "logging.file";
+
+	/**
+	 * The name of the Spring property that contains the directory where log files are
+	 * written.
+	 */
+	@Deprecated
+	public static final String DEPRECATED_PATH_PROPERTY = "logging.path";
```

Comment:
I think this should be named FILE_PATH_PROPERTY, should not be deprecated, and its value should be `logging.file.path`. It should be documented as `@since 2.2.0`.

Reply:
Done